docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs

```diff
@@ -45,7 +45,7 @@ public static void Example()
             // Load the TensorFlow model once.
             //      - Use it for quering the schema for input and output in the model
             //      - Use it for prediction in the pipeline.
-            var modelInfo = TensorFlowUtils.LoadTensorFlowModel(mlContext, modelLocation);
+            var modelInfo = mlContext.Transforms.LoadTensorFlowModel(modelLocation);
```

Comment:
>mlContext.Transforms.LoadTensorFlowModel [](start = 28, length = 40)

This doesn't seem like a transform. This seems like you're loading a model, so should it be under "model?"

Reply:
Sounds reasonable. I see some of the ONNX model related methods are available via `model` catalog. I will move it. Thanks!

---
In reply to: [259530418](https://github.com/dotnet/machinelearning/pull/2672#discussion_r259530418) [](ancestors = 259530418)